src/backend/metal/src/device.rs

```diff
@@ -1910,7 +1915,7 @@ impl hal::Device<Backend> for Device {
 
         if swizzle != format::Swizzle::NO {
             error!("swizzling not supported");
-            return Err(image::ViewError::Unsupported);
+            //return Err(image::ViewError::Unsupported);
```

Comment:
Do we need to leave this out?

Reply:
I don't think swizzling should be the end of the world. We've yet to figure out how to properly limit this with VkPI group, so just commenting the return out seems fine for now.